apps/web/src/pages/api/trpc/[trpc].ts

```diff
@@ -2,10 +2,17 @@ import { appRouter, createContext } from "@dotkomonline/gateway-trpc"
 import { createNextApiHandler } from "@trpc/server/adapters/next"
 import { applyCorsMiddleware } from "@/middlewares/cors"
 
+import { getLogger } from "@dotkomonline/logger"
+
+export const logger = getLogger("migrator")
```

Comment:
change name

Reply:
I have not looked deeply into winston logging, but i did not get the output I wanted when I tried to use it. I found it very useful to add the console.error in the onError to detect errors from trpc during development, but i'm not sure if this should be pushed to main